src/Codeception/Module/Symfony.php

```diff
@@ -284,11 +296,13 @@ protected function getKernelClass()
 
         require_once $file;
 
-        $possibleKernelClasses = [
-            'AppKernel', // Symfony Standard
-            'App\Kernel', // Symfony Flex
-        ];
-        foreach ($possibleKernelClasses as $class) {
+        // mixing default kernel classes and class with changed namespace
+
+        if ($this->config['app_kernel']) {
+            $this->kernelClasses = array_merge($this->kernelClasses, [$this->config['app_kernel']]);
```

Comment:
If we already declared class name explicitly name in config why would we need to look for other kernels?
Why not just `$this->kernelClasses = [$this->config['app_kernel']`

Reply:
Ok it seems reasonable, i've mixed it just in case the "fool's protection". If u're insisting on that change as it can have an impact on speed it's not problems to fix it =) I also thought of some refactoring for this module cause it looks pretty dirty. Does the module maintainer is actual or u're open to add some maintainers there?